src/store/ChartStore.js

```diff
@@ -284,55 +282,51 @@ class ChartStore {
      * Get tradeTimes if not loaded yet
      * OR update the active symbols by comapring open time
      */
-    @action.bound updateMarketClosedStatus() {
+    onMarketClosedStatus() {
+        this.api.getTradingTimes()
+            .then(this.updateMarketClosedStatus)
+            .catch(data => console.warn(data.error));
```

Comment:
@brucebinary I don't know If this line is necessary or should be removed

Reply:
@shayanbinary should be fine if you remove it. I don't really have any preference about it